commands/eval.b.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ * Command(s) for evaluating PHP code directly.
+ */
+
+/**
+ * Implements hook_b_command().
+ */
+function eval_b_command() {
+  return array(
+    'php-eval' => array(
+      'description' => bt('Evaluate arbitrary PHP code after bootstrapping Backdrop.'),
+      'callback' => 'php_eval_b_command',
+      'arguments' => array(
+        'code' => bt('The PHP code to evaluate.'),
+      ),
+      'aliases' => array('ev', 'eval'),
+      'bootstrap' => B_BOOTSTRAP_FULL,
+      'examples' => array(
+        'b php-eval \'$node = node_load(1); print $node->title;\'' => 'Loads node with nid 1 and then prints its title.',
+        'b php-eval "node_access_rebuild();"' => 'Rebuild node access permissions.',
+        'b php-eval "file_unmanaged_copy(\'$HOME/Pictures/image.jpg\', \'public://image.jpg\');"' => 'Copies a file whose path is determined by an environment\'s variable. Note the use of double quotes so the variable $HOME gets replaced by its value.',
+      ),
+    ),
+  );
+}
+
+/**
+ * Command callback: Execute specified PHP code.
```

Comment:
The description after the colon should be an exact copy of the command description (see other commands for examples).

Reply:
Done.

Reply:
Thanks, but this'll still need to be updated when the description is updated with the bracketed text.

Reply:
(run/execute) added to PR in both places.